integration-tests/plugin-tests/src/test/kotlin/com/r3/conclave/integrationtests/general/tests/plugin/CopyGraalVMTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.r3.conclave.integrationtests.general.tests.plugin
+
+import com.r3.conclave.integrationtests.general.commontest.TestUtils.RuntimeType.GRAALVM
+import java.nio.file.Path
+import kotlin.io.path.div
+
+/**
+ * This runs the tests from [AbstractPluginTaskTest], in particular making sure it's incremental (i.e. doesn't run
```

Comment:
This is a well-known behaviour of Junit whereby it will execute tests in super classes first. It does look odd that this class has not `@Test` methods, but that's why I have this comment.

Reply:
Yes, that was my intuition, even though I find it a bit obscure. I.e. we override some parameters but we do not see even any code that trigger those tests, relying on the Junit mechanism that runs tests of the base class.